README.md

```diff
@@ -15,7 +15,8 @@ Other main reasons for the development of this tool have been the transparent su
 ## Install
 
 ```bash
-$ pip install azmlops
+$ git clone git@github.com:JacopoMangiavacchi/azmlops.git
```

Comment:
Have you published changes on PyPl or TestPyPl ?  Why in case you have to also clone repo and pip install from that ?  I think if you want to "manually" install the package you should run ```python setup.py install```.  Isn't it ?

Reply:
Hey Jacopo, here's what I get when trying to install from PyPI:

```
azmlops git:main ❯ pip3 install azmlops                                                                                                                                                                
ERROR: Could not find a version that satisfies the requirement azmlops
ERROR: No matching distribution found for azmlops
```

Is it published under a different name or something?

Reply:
I can also change the command for local install (which I think is fine if it's not ready/intended for public release). This does work though.